libs/application/template-api-modules/src/lib/modules/templates/financial-statements-inao/financial-statements-inao.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common'
+import { FinancialStatementsInaoClientService } from '@island.is/clients/financial-statements-inao'
+import * as kennitala from 'kennitala'
+
+import { TemplateApiModuleActionProps } from '../../../types'
+
+@Injectable()
+export class FinancialStatementsInaoTemplateService {
+  constructor(
+    private financialStatementsService: FinancialStatementsInaoClientService,
+  ) {}
+
+  async getUserType({ auth }: TemplateApiModuleActionProps) {
+    const { nationalId } = auth
+
+    if (kennitala.isPerson(nationalId)) {
+      return this.financialStatementsService.getClientType('Einstaklingur')
```

Comment:
Put 'Einstaklingur' in a constant, shareable with front end?

Reply:
Will probably add this to environment variable in later branch